deploy/bash/generate-sa-kubeconfig.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+read -p "Enter Kube Api Server: " server
+read -p "Enter Kube Context: " ctx
```

Comment:
Couldn't we just hardcode a value for the context name? I don't think it matters at all?